src/odemis/gui/win/acquisition.py

```diff
@@ -1033,6 +1081,28 @@ def _fit_view_to_area(self):
         fov = (self.area[2] - self.area[0], self.area[3] - self.area[1])
         self.pnl_view_acq.set_mpp_from_fov(fov)
 
+    def _get_areas(self) -> List[Tuple[float]]:
+        """
+        return: the bounding boxes (in physical coordinates) of all areas to acquire
+        """
+        if not self._main_data_model.sample_centers:
+            return [self.area]
+
+        # TODO: for now this is only for the MIMAS, but eventually, on the METEOR,
+        # which often supports 2 grids, this should also be possible to use.
+
+        # .sample_centers contains the center position of each area
+        # .sample_rel_bbox contains the relative bounding box on an area
+        rel_bbox = self._main_data_model.sample_rel_bbox
+
+        # TODO: once there are toggle buttons, filter to only the button toggled on
+        # Sort the centers along X, and for centers with the same X, along the Y.
```

Comment:
is this the order in which they will be acquired?

Reply:
Indeed. The actual order doesn't matter much (maybe just saves a few seconds of stage movement at best), but what's important is that it's always the same order, to avoid "surprises" to the user. I'll add this to the comment.